readme.MD

```diff
@@ -1,16 +1,47 @@
 # Dialogs Framework
 The Dialogs Framework is a Python library for writing resumable functions with persistent execution state.
```

Comment:
```suggestion
A Python library for creating resumable dialog interfaces
```

Reply:
Accepted but rephrased, I prefer not to use "dialog" in the definition